modules/promotions/admin-menu-items/go-pro-promotion-item.php

```diff
@@ -3,14 +3,19 @@
 namespace Elementor\Modules\Promotions\AdminMenuItems;
 
 use Elementor\Core\Admin\Menu\Interfaces\Admin_Menu_Item_With_Page;
+use Elementor\Core\Utils\Promotions\Validate_Promotion;
 use Elementor\Settings;
 
 if ( ! defined( 'ABSPATH' ) ) {
 	exit; // Exit if accessed directly
 }
 
 class Go_Pro_Promotion_Item implements Admin_Menu_Item_With_Page {
-	const URL = 'https://go.elementor.com/pro-admin-menu/';
+	CONST URL = 'https://go.elementor.com/pro-admin-menu/';
```

Comment:
```suggestion
	const URL = 'https://go.elementor.com/pro-admin-menu/';
```
There does not need to be a change here.